src/Features/ERDDAP/Platform/Observations/CurrentConditions/wind.tsx

```diff
@@ -24,17 +24,22 @@ export const WindCard: React.SFC<Props> = ({ datasets, unit_system }) => {
     return null
   }
 
-  const data: DataTimeSeries[] = datasets.map(dataset => ({
+  const data: DataTimeSeries[] = datasets.map((dataset) => ({
     name: dataset.data_type.long_name,
     timeSeries: dataset.readings,
-    unit: dataset.data_type.units
+    unit: dataset.data_type.units,
   }))
 
-  const speed = datasets.filter(dataset => dataset.data_type.standard_name.includes("speed"))
-  const gust = datasets.filter(dataset => dataset.data_type.standard_name.includes("gust"))
-  const direction = datasets.filter(dataset => dataset.data_type.standard_name.includes("direction"))
+  const speed = datasets.filter((dataset) => dataset.data_type.standard_name.includes("speed"))
+  const gust = datasets.filter((dataset) => dataset.data_type.standard_name.includes("gust"))
+  const direction = datasets.filter((dataset) => dataset.data_type.standard_name.includes("direction"))
 
-  const data_converter = converter([...speed, ...gust][0].data_type.standard_name)
+  const speedAndGust = [...speed, ...gust]
+  if (speedAndGust.length < 1) {
+    return null
+  }
+
+  const data_converter = converter(speedAndGust[0].data_type.standard_name)
```

Comment:
Codacy found an issue: [Identifier 'data_converter' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6084269)